llvm/lib/Target/SystemZ/SystemZISelLowering.cpp

```diff
@@ -8113,10 +8583,151 @@ static bool combineCCMask(SDValue &CCReg, int &CCValid, int &CCMask) {
     CCReg = IPM->getOperand(0);
     return true;
   }
-
   return false;
 }
 
+std::optional<SDValue>
```

Comment:
As an aside: `SDValue` has a built-in default value `SDValue()`, so I don't think the `std::optional` is needed here.